### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.iothubprovisioningservices.models.error_messsage.ErrorMesssage
  fullName: azure.mgmt.iothubprovisioningservices.models.error_messsage
  langs:
  - python
  module: azure.mgmt.iothubprovisioningservices.models.error_messsage
  name: error_messsage
  source:
    id: error_messsage
    path: azure-mgmt-iothubprovisioningservices\azure\mgmt\iothubprovisioningservices\models\error_messsage.py
    remote:
      branch: master
      path: azure-mgmt-iothubprovisioningservices\azure\mgmt\iothubprovisioningservices\models\error_messsage.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 0
  type: module
  uid: azure.mgmt.iothubprovisioningservices.models.error_messsage
references:
- fullName: azure.mgmt.iothubprovisioningservices.models.error_messsage.ErrorMesssage
  isExternal: false
  name: ErrorMesssage
  parent: azure.mgmt.iothubprovisioningservices.models.error_messsage
  uid: azure.mgmt.iothubprovisioningservices.models.error_messsage.ErrorMesssage
